addons/api/tests/unit/serializers/target-test.js

```diff
@@ -143,6 +146,76 @@ module('Unit | Serializer | target', function (hooks) {
       session_max_seconds: 28800,
       session_connection_limit: null,
       worker_filter: 'worker',
+      egress_worker_filter: null,
+      ingress_worker_filter: null,
+      attributes: {
+        default_port: 1234,
+      },
+    });
+  });
+
+  test('it serializes the egress_worker_filter attribute if present', function (assert) {
+    assert.expect(1);
+    const store = this.owner.lookup('service:store');
+    const serializer = store.serializerFor('target');
+    const record = store.createRecord('target', {
+      name: 'User',
+      description: 'Description',
+      version: 1,
+      type: TYPE_TARGET_SSH,
+      egress_worker_filter: 'egress worker',
+      default_port: 1234,
+      scope: {
+        scope_id: 'org_1',
+      },
+    });
+    const snapshot = record._createSnapshot();
+    const serializedRecord = serializer.serialize(snapshot);
+    assert.deepEqual(serializedRecord, {
+      name: 'User',
+      description: 'Description',
+      version: 1,
+      type: TYPE_TARGET_SSH,
+      scope_id: 'org_1',
+      session_max_seconds: 28800,
+      session_connection_limit: null,
+      worker_filter: null,
```

Comment:
If worker wasn't set to begin with, should it be serialized?  May want to follow up with BE.

Reply:
Followed up with the BE team. `worker_filter` being present and set to null should not be an issue.